indra/llcommon/llapr.cpp

```diff
@@ -606,7 +607,11 @@ S32 LLAPRFile::writeEx(const std::string& filename, const void *buf, S32 offset,
         apr_status_t s = apr_file_write(file_handle, buf, &bytes_written);
         if (s != APR_SUCCESS)
         {
-            LL_WARNS("APR") << " Attempting to write filename: " << filename << LL_ENDL;
+            LL_WARNS("APR") << "Attempting to write filename: " << filename << LL_ENDL;
+            if (APR_STATUS_IS_ENOSPC(s))
+            {
+                LLApp::error("There is not enough space on the disk", false);
```

Comment:
you can use LLUserWarningMsg::show(const std::string& message)

Reply:
> But the bigest proble: this is going to spam, viewer handles a lot of textures.

LLAppViewer::showError proposes to terminate the program in case of non-critical error
User can press OK if these message become bothering him and terminate the viewer

Reply:
It's called from a thread, there is more than one.